app/tasks/task6/form.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Form</title>
+
+</head>
+<!-- Shift + Alt + F for formatting the code -->
+
+<body>
+    <h1>First form</h1>
+
+    <form action="#" method="post">
+        <fieldset>
+            <legend>Personal Info</legend>
+            <div>
+                <label for="name"> First Name:</label>
```

Comment:
``for="fname"`` ar trebui sa fie: valoarea id tre sa coincinda cu attributul for la label

Reply:
fixed, for="fname", id="fname" and name="fname", the same was fixed for "lname"